app/main.py

```diff
@@ -1 +1,75 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name):
+        self.name = name
+
+    @classmethod
+    def learn_skill(cls, skill):
+        cls.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = [
+        "JavaScript",
+        "HTML",
+        "CSS"
+    ]
```

Comment:
I believe that in this task, the skills should be stored in the attributes of the constructor, not the class, since the examples assume that the instances of these classes are a specific programmer with his skills, and not the position on which he works, if you want to add or remove a skill, you remove it for all programmers, since they will refer to a common attribute, and not their own

Reply:
Yes, i agree. It was obvious to provide that list for every developer, but i didn`t notice that. Changes provided.